cluster/swarm/node.go

```diff
@@ -265,7 +265,7 @@ func (n *node) updateContainer(c dockerclient.Container, containers map[string]*
 		}
 		container.Info = *info
 		// real CpuShares -> nb of CPUs
-		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 100.0 * n.Cpus
+		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 1024.0 * n.Cpus
```

Comment:
@smothiki right, I thought using 1024.0 was enough but as `container.Info.Config.CpuShares` is an int it doesn't work, I'll change thanks.